nncf/experimental/openvino_native/quantization/algorithms/fast_bias_correction/openvino_backend.py

```diff
@@ -87,11 +87,11 @@ def get_sub_input_output_names(subgraph: ov.Model) -> Tuple[str, str]:
         return subgraph.inputs[0].node.friendly_name, subgraph.outputs[0].node.friendly_name
 
     @staticmethod
-    def create_blob(shape: Tuple[int], data: List[float]) -> np.ndarray:
-        blob = np.zeros(shape)
-        for i, value in enumerate(data):
-            blob[:, i] = value
-        blob = blob.astype(np.float32)
+    def create_blob(shape: Tuple[int], data: List[float], channel_axis: int) -> np.ndarray:
+        blob = np.zeros(shape, dtype=np.float32)
```

Comment:
It looks like the data type depends on `data`. I'm surprised that only `float` is supported here. What about FP16 and etc.?

cc' @KodiaqQ

Reply:
@KodiaqQ do you suggest to add `model` in a signature of create_blob?

Reply:
Not, I'm not. I suggest using the data type of the input data and verifying these changes on the full FP16 model.

Reply:
Removed dtype param as it is done in POT.

Reply:
So removing wasn't possible there as during inference for ONNX it demands the correct input dtype. So, decided to use the same dtype for filling blob as input data tensor.